<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeBlog</class>
 <widget class="QDialog" name="MakeBlog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MakeBlog</string>
  </property>
  <widget class="QTextEdit" name="inputText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>631</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="copybtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>121</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>내용복사</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textLink">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>링크주소를 입력하세요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBlog">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>블로그 텍스트 추출</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="ageRadio0">
      <property name="text">
       <string>전연령</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ageRadio1">
      <property name="text">
       <string>20대</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ageRadio2">
      <property name="text">
       <string>30대</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ageRadio3">
      <property name="text">
       <string>40대</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>107</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>결과보기</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QRadioButton" name="jobRadio0">
      <property name="text">
       <string>전문가</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="jobRadio1">
      <property name="text">
       <string>리뷰어</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="jobRadio2">
      <property name="text">
       <string>마케터</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="jobRadio3">
      <property name="text">
       <string>블로거</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
